examples/expressions/10_expressions_intro.py

```diff
@@ -2,49 +2,58 @@
 .. _example_expressions_intro:
 
 
-Building complex tabular pipelines
+Building a predictive model by combining multiple tables with the skrub Data Plan
 ==================================
```

Comment:
```suggestion
Building a predictive model by combining multiple tables with the skrub Data Plan
====================================================================
```
I think that the line must be longer, to avoid invalid rst